### PR TITLE
fix/ added mock service and deleted old one

### PR DESCRIPTION
**Pull request by NurmukhammadMakhmudov**

### CODE REVIEW

File: src/app/models/admin-profile/admin-profile.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import { ServiceComponent } from './service/service.component';
```
Comment (RomPavel) [inline src/app/models/admin-profile/admin-profile.component.ts:2]: do you need this

File: src/assets/adminLocalStorage.json [hunk 2]
```diff
@@ -0,0 +1,1102 @@
+[
```
Comment (RomPavel) [inline src/assets/adminLocalStorage.json:1]: its should be in mocks not assets

File: src/app/models/admin-profile/service/service.component.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+})
+export class ServiceComponent {
+  constructor(private readonly http: HttpClient) {}
+
+  getData(): Observable<TestData[]> {
+    return this.http.get<TestData[]>(URL);
+  }
+
+  postData(data: TestData): Observable<TestData> {
+    // @ts-ignore
```
Comment (RomPavel) [inline src/app/models/admin-profile/service/service.component.ts:19]: remove this

File: src/app/models/admin-profile/service/service.component.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { TestData } from '../../../../mocks/admin-profile-utils.mock';
+
+const URL = '../../../assets/adminLocalStorage.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ServiceComponent {
```
Comment (RomPavel) [inline src/app/models/admin-profile/service/service.component.ts:11]: ServiceComponent naming is too abstract, and its not a component

File: src/app/models/admin-profile/service/service.component.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+  getData(): Observable<TestData[]> {
+    return this.http.get<TestData[]>(URL);
+  }
+
+  postData(data: TestData): Observable<TestData> {
+    // @ts-ignore
+    return this.http
+      .post<TestData>('http://httpbin.org/post', data)
+      .toPromise()
```
Comment (RomPavel) [inline src/app/models/admin-profile/service/service.component.ts:22]: lets not promisify requests

File: src/app/models/admin-profile/service/service.component.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+const URL = '../../../assets/adminLocalStorage.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ServiceComponent {
+  constructor(private readonly http: HttpClient) {}
+
+  getData(): Observable<TestData[]> {
+    return this.http.get<TestData[]>(URL);
```
Comment (RomPavel) [inline src/app/models/admin-profile/service/service.component.ts:15]: in our approach, we need to have store service as well
check auth-store.service or test-store.service

File: src/app/models/admin-profile/admin-table/table.component.ts [hunk 7]
```diff
@@ -65,10 +80,10 @@ export class TableComponent implements OnInit {
     this.AssignSelector = !this.AssignSelector;
     if (this.AssignSelector) {
       this.displayedColumns = ['ID', 'Position', 'Level', 'Date', 'Coach', 'Button'];
-      this.data = this.assignedTests;
+      this.dataPrototype = this.assignedTests;
     } else {
       this.displayedColumns = ['ID', 'Position', 'Level', 'Date', 'Button'];
-      this.data = this.notAssignedTests;
+      this.dataPrototype = this.notAssignedTests;
```
Comment (RomPavel) [inline src/app/models/admin-profile/admin-table/table.component.ts:86]: dataPrototype is super unclear naming

File: src/app/models/admin-profile/service/service.component.ts (hunk 4 above)
Comment (Inunor) [reply src/app/models/admin-profile/service/service.component.ts:11]: function naming as well, I think getTests would be much better then getData. Data doesn't give any information -> data is everything and nothing)

**Review by Inunor — CHANGES_REQUESTED** [review]

File: src/app/models/admin-profile/admin-profile.component.ts [hunk 8]
```diff
@@ -1,10 +1,35 @@
...
+
+  adminTable: { [key: string]: TestData[] } = {};
+
+  adminDisplayListMap: { [key: string]: string[] } = {};
+
   ngOnInit(): void {}
+
+  constructor(private service: ServiceComponent, private service2: AdminTableStoreService) {
+    this.service2.getTestData().subscribe((data) => {
+      console.log(data);
```
Comment (ExxZzyy) [inline src/app/models/admin-profile/admin-profile.component.ts:22]: Remove console.log

File: src/app/models/admin-profile/service/service.component.ts [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+const URL = '../../../assets/admin.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ServiceComponent {
+  constructor(private readonly http: HttpClient) {}
+
+  getData(): Observable<TestData[]> {
+    console.log(this.http.get<TestData[]>(URL));
```
Comment (ExxZzyy) [inline src/app/models/admin-profile/service/service.component.ts:15]: Remove console.log

**Review by ExxZzyy — APPROVED** [review]

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**